README.md

```diff
@@ -14,14 +14,29 @@ Once you've created something amazing with Open MCT, showcase your work in our G
 Building and running Open MCT in your local dev environment is very easy. Be sure you have [Git](https://git-scm.com/downloads) and [Node.js](https://nodejs.org/) installed, then follow the directions below. Need additional information? Check out the [Getting Started](https://nasa.github.io/openmct/getting-started/) page on our website.
 (These instructions assume you are installing as a non-root user; developers have [reported issues](https://github.com/nasa/openmct/issues/1151) running these steps with root privileges.)
 
-1. Clone the source code `git clone https://github.com/nasa/openmct.git`
+1. Clone the source code 
```

Comment:
Let's add `:` to the end of each of these numbered steps

Reply:
could you commit suggestion?

Reply:
Sure, are you unable due to permissions?

Reply:
Sure